.gitignore

```diff
@@ -10,3 +10,6 @@
 
 # rspec failure tracking
 .rspec_status
+
+# RubyMine files
+/.idea/*
```

Comment:
shouldn't people just be putting this in their global .gitignore

Reply:
I don't feel too strongly on this either way. We have it in ez-rails (not the strongest argument, I know) and it seems like a good insurance policy against people accidentally committing these files if they don't have it in a global .gitignore.

Reply:
Removed this in the interest of getting these changes completed.